inst/extdata/aglu/Muller_crops.csv

```diff
@@ -0,0 +1,21 @@
+# File: Muller_crops.csv,
```

Comment:
See comment below--let's straighten out source and then ensure filename consistent with it.